src/partials/reviews.html

```diff
@@ -0,0 +1,110 @@
+<section id="reviews">
+  <div class="container">
+    <h2 class="section-heading">
+      What our <span class="brand-underline">students</span> say
+    </h2>
+    <p class="secondary-text">
+      Hear from our students about their learning journey with us, as they share
+      their experiences, progress, and the transformative impact
+    </p>
+    <ul class="review-cards-list">
+      <li class="review-card">
+        <div class="review-header">
+          <picture>
+            <source
+              srcset="
+                images/reviews/anna-des.png    1x,
+                images/reviews/anna-des@2x.png 2x
+              "
+              media="(min-width: 1280px)"
```

Comment:
we are using 1152 in CSS, maybe we should use that value here too?

Reply:
Yep, i found that we using 1152 later... Fixed